__tests__/app.test.js

```diff
@@ -51,39 +51,112 @@ describe("app.js", () => {
                   title: expect.any(String),
                   topic: expect.any(String),
                   author: expect.any(String),
-                  body: expect.any(String),
                   created_at: expect.any(String),
                   votes: expect.any(Number),
                 })
               );
+              expect(article.hasOwnProperty("body")).toBe(false);
```

Comment:
A Nice addition here - I may generally recommend that we only assert the existence of properties that _**_**should**_**_ be there - just in case you add it in for example at a later point